src/e-init.js

```diff
@@ -182,6 +182,13 @@ program
       const opts = { stdio: 'inherit' };
       childProcess.execFileSync(process.execPath, [e, 'use', name], opts);
 
+      // ensure xcode is loaded
+      if (process.platform === 'darwin') {
+        loadXcode(true);
+      }
+
+      ensureRoot(config, !!options.force);
+
```

Comment:
FYI: I structured the functions this way because `loadXcode` has to come after the saving of the config and once `e use` has been called because it uses the config. `ensureRoot` relies on xcode so it comes afterward. Let me know if this looks correct or not!